templates/layouts/eu_footer_middle.html.twig

```diff
@@ -1,4 +1,4 @@
-<h4> {{ contact_eu_title|t }}  </h4>
+<p class="fw-bold border-bottom pb-2 mb-2"> {{ contact_eu_title|t }}  </p>
```

Comment:
Please make a test, isn't contact_eu_title already sent translated (if the translation is added ?), same for below ones